.github/workflows/operator-ui-ci.yml

```diff
@@ -6,6 +6,26 @@ env:
   TARGET_BRANCH_NAME: ${{ github.event.pull_request.base.ref }}
 
 jobs:
+  changes:
```

Comment:
This is adding needless complexity to this pipeline. This job takes 1.5m to run.